euclidlike_imsim/noise.py

```diff
@@ -97,8 +97,15 @@ def get_noise(cfg_noise, cfg_image, base, logger):
     # Make integer ADU now.
     noise_img.quantize()
 
+    # Add quantization noise (this avoid issues related to rounding e.g.
+    # backgroud estimation)
+    quantization_noise = noise_img.copy()
+    quantization_noise.fill(0)
+    quantization_noise.addNoise(galsim.DeviateNoise(galsim.UniformDeviate(rng)))
+    noise_img += quantization_noise
+    noise_img -= 0.5
```

Comment:
Where does this subtraction by 0.5 come from?

Reply:
With quantization we assume that if you have a value of 100.5 ADUs it has the same chance to round to 100 or 101 which lead to +/-0.5. The Uniform distribution from GalSim return value in the range [0, 1], the -0.5 is just to rescale the range to [-0.5, 0.5].